src/use-dropdown-menu.ts

```diff
@@ -233,5 +234,5 @@ export default function useDropdownMenu(itemCount: number, options?: DropdownMen
 	}));
 
 	// Return a listener for the button, individual list items, and the state of the menu
-	return { buttonProps, itemProps, isOpen, setIsOpen } as const;
+	return { buttonProps, itemProps, isOpen, moveFocus, setIsOpen } as const;
```

Comment:
Similarly, I would swap `moveFocus` with `setIsOpen` so that the two "is open" related variables are grouped together.

```suggestion
	return { buttonProps, itemProps, isOpen, setIsOpen, moveFocus } as const;
```